.github/workflows/release.yaml

```diff
@@ -0,0 +1,44 @@
+name: NATS Server Releases
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  run:
+    name: GitHub Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: true
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "stable"
```

Comment:
Should we add a comment clarifying that:
`# Note that the version we set up here does not matter, since we specify GOTOOLCHAIN in goreleaser.yml`

Reply:
The `toolchain` is a minimum, if `stable` surpasses `toolchain` then `stable` will be used instead.